pkg/kubectl/cmd/cmd.go

```diff
@@ -112,6 +112,8 @@ __custom_func() {
 `
 	valid_resources = `Valid resource types include:
    * componentstatuses (aka 'cs')
+   * daemonsets (aka 'ds')
+   * deployments
```

Comment:
Thanks. Could you also add an abbreviation for deployments? I propose 'dep'.

cc @kargakis @janetkuo @mqliang 

expandResourceShortcut:
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/kubectl.go#L97

Really, we should autogenerate this documentation and that map from the same source. We can do that later, but how about adding comments to both places? I guess here it would have to go before or after this string.